packages/sanity/src/_singletons/core/form/VersionContext.tsx

```diff
@@ -4,14 +4,15 @@
 // eslint-disable-next-line import/consistent-type-specifier-style
 import {createContext, type ReactElement} from 'react'
 
-import type {Version} from '../../../core/versions/types'
-import {BUNDLES, LATEST} from '../../../core/versions/util/const'
+import {useBundlesStore} from '../../../core/store/bundles'
```

Comment:
Maybe it's part of a future change, but just asking to keep it consistent with the rest of contexts added here.

What I see in the rest of files in `_singletons` is that this only haves the context creation.

see for example https://github.com/sanity-io/sanity/blob/corel-35/packages/sanity/src/_singletons/core/form/FormValueContext.ts

Will it make sense to move the rest of things in this folder out to a VersionProvider ?

Also, in the provider or here, could we add a `useVersion` hook that exposes the context like https://github.com/sanity-io/sanity/blob/corel-35/packages/sanity/src/core/store/bundles/BundlesProvider.tsx#L41-L47

Reply:
We had a discussion off github and it makes more sense to move all this logic to the hook (since it doesn't need to be a context, that will happen in a separate PR)